model/Post.php

```diff
@@ -0,0 +1,110 @@
+<?php 
+
+// Define namespace of this class :
+namespace EmilieSchott\BlogPHP\Model;
+
+// Import Emilie Schott other class needed into the global namespace :
+use EmilieSchott\BlogPHP\Model\Hydrate;
+
+// Load other class needed:
+require_once __DIR__ . '/Hydrate.php';
+
+// Define this class :
+class Post extends Hydrate {
+
+    // Constants : 
+    private $id;
+    private $usersId;
+    private $title;
+    private $standFirst;
+    private $content;
+    private $createdAt;
+    private $updatedAt;
+    private $author;
+    private $picture;
+
+    // Getters :
```

Comment:
For better reading and coding conventions, your getters should be name `getXxx` (or `isXxx` if the property is a boolean).

Reply:
ok! Copy that too. I'm going to make the changes.